docs/deployment.md

```diff
@@ -0,0 +1,40 @@
+<!---
+  Copyright (c) 2021 Robert Bosch GmbH
+
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+  SPDX-License-Identifier: MPL-2.0
+-->
+
+# Deployment of Vehicle Edge
+
+## Overview
+
+The Vehicle Vehicle Edge consists out of several docker containers to setup the mandatory platform and platform services and an optional vehicle integration for development and testing purpose.
+
+## Deployment Diagram
+
+![./artifacts/plantuml/vehicle_edge_deployment.svg](./artifacts/plantuml/vehicle_edge_default_deployment.svg)
+
+This diagram shows the 'default' deployment of Vehicle Edge when the docker-compose file _docker-compose.edge.yml_ is used. It shows only one possible for a deployment and should work as an example how it could look like. The packages are used to logically group the docker-containers regarding their scope and feature set.
```

Comment:
_possibility_ for a deployment or "possible deployment"

Reply:
Done